allium/templates/network-health-dashboard.html

```
@@ -377,21 +377,83 @@ <h4><a href="#operator-participation" class="al-link-plain">👥 Operator Partic
+                        <span class="metric-value">{{ "{:,}".format(relays.json.network_health.uri_familyid_ed25519_valid) }}/{{ "{:,}".format(relays.json.network_health.uri_familyid_ed25519_total) }} ({{ "%.1f%%"|format(relays.json.network_health.uri_familyid_ed25519_success_rate) }})</span>
+                        <span class="metric-label">URI-FamilyID <span style="font-size: 9px; padding: 0 4px; background-color: #007bff; color: white; border-radius: 2px;">v3</span></span>
+                    </div>
+                </div>
+                {% endif %}
+
+                <!-- B2.2 (revised per UX feedback): ciissversion adoption
+                     as 3 separate metric tiles matching the standard
+                     dashboard pattern (bold-black-value on top, grey
+                     label below). -->
+                {% set _decl = relays.json.network_health.ciissversion_declared or {} %}
+                {% if _decl.get('3', 0) > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)